.circleci/config.yml

```diff
@@ -22,7 +22,14 @@ jobs:
       - run: make deps DEPARGS="-v"
       - run: make errcheck
       - run: make check-docs
-      - run: make format
```

Comment:
Can we quickly add proto as well with  `make format proto` as well? Quick win? (:

Reply:
ok no problem, I forgot about it sorry

Reply:
Added as separate stage if it doesn't bother you. It feels cleaner to me that user see exactly which stage failed.